### ISSUE TITLE
Shared index style

### ISSUE THREAD

**Issue by pimentel** [pachterlab/bears_analyses#8]
A few things about the shared index:
- When looking at the urls in `species.json`, I don't seem to see the actual url of the file. Are you prefixing it somewhere with the ensembl the url? It would be best to simply have a generic url for this
- The kmer length should be included in the index file name. To be clear, you can have 1 fasta to many indices

That's all I can think of for now. Will look at it more closely later.

**Comment by psturmfels** [comment]
Oh woah made a big mistake with the urls. On it.

**Comment by psturmfels** [comment]
Url problem is fixed. I'll take a look at adding kmer length later today.

**Comment by pimentel** [comment]
Cool, thanks. Also, I feel like `species.json` should really be `species.py`. The reason being is the following:

``` python
ensembl_url = 'http://...'
species = {'homo_sapien': ensembl_url + '/....fa', 'mus_musculus': ensembl_url + '/...fa'}
```

which you cannot do in json (because you can't have "variables" that are referenced with in the json file).

This pattern is helpful because if we decide to simply update to a new version of ensembl, all you have to do is change the main url.

Then, in the main `Snakefile` you simply do the following:

```
include: 'species.py'
```

then you reference the `species` dictionary

**Comment by psturmfels** [comment]
A fair point. This will be incorporated tomorrow.

**Comment by psturmfels** [comment]
Closed with fa5eec6a7931c18b80198aa5922ce88f2da38eef
